### PR TITLE
Two pointer documentation

### LINKED ISSUE #75: Article for Two pointer Algorithm.
## 📚 DESCRIPTION

I want to explain the usage of two pointer algorithm in the article according to these points:-
1.how this work..?
2. What will be time and space complexity?
3. Why we use this algorithm?
4. Where we can use it?
5. Key points about algorithm..
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

### PR DESCRIPTION
**Pull request by RuchiGitHu**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes
### Description

I want to explain the usage of two pointer algorithm in the article

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

closes #75

### CODE REVIEW

**Comment by RuchiGitHu** [comment]
Please review @HarshCasper @ankitaggarwal23

**Comment by RuchiGitHu** [comment]
> Please review @HarshCasper @ankitaggarwal23

can anyone review please... @HarshCasper @ankitaggarwal23

**Comment by RuchiGitHu** [comment]
> Please review @HarshCasper @ankitaggarwal23

Please review @HarshCasper

File: sidebars.js [hunk 1]
```diff
@@ -17,6 +17,13 @@ module.exports = {
                 "code-playgrounds/java-playground",
             ],
         },
+        {
+            type: "category",
+            label: "Algorithms",
+            items: [
+                "Algorithms/Two_Pointer",
+            ],
+        },
```
Comment (HarshCasper) [inline sidebars.js:26]: Why is this needed? We have already segregated algorithms.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Fix the CI build as well. Do a `yarn build` on your local machine and push the updates only when they pass.

**Comment by RuchiGitHu** [comment]
> Fix the CI build as well. Do a `yarn build` on your local machine and push the updates only when they pass.

Please review @HarshCasper .
One thing I want to say that NeoAlgo is just awesome. Thanks for this chance to contribute here, because of this i have build yarn first time.
Thanks again..
Looking forward to contribute here.

**Comment by RuchiGitHu** [comment]
> > Fix the CI build as well. Do a `yarn build` on your local machine and push the updates only when they pass.
> 
> Please review @HarshCasper .
> One thing I want to say that NeoAlgo is just awesome. Thanks for this chance to contribute here, because of this i have build yarn first time.
> Thanks again..
> Looking forward to contribute here.

Please review @HarshCasper

**Comment by RuchiGitHu** [comment]
> > Fix the CI build as well. Do a `yarn build` on your local machine and push the updates only when they pass.
> 
> Please review @HarshCasper .
> One thing I want to say that NeoAlgo is just awesome. Thanks for this chance to contribute here, because of this i have build yarn first time.
> Thanks again..
> Looking forward to contribute here.

Please review @HarshCasper

**Comment by RuchiGitHu** [comment]
Please review @ankitaggarwal23

**Review by iamrajiv — APPROVED** [review]

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**